staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -762,6 +774,16 @@ func (s unstructuredNegotiatedSerializer) SupportedMediaTypes() []runtime.Serial
 			EncodesAsText:    true,
 			Serializer:       json.NewYAMLSerializer(json.DefaultMetaFactory, s.creator, s.typer),
 		},
+		{
```

Comment:
would it be clearer to put some sort of attribute on SerializerInfo indicating it can only be used for specific types, and then check in `acceptMediaTypeOptions` in the non-conversion case if the incoming `accepts` info allows an unconverted use?

Reply:
Then we have to put two serializer infos with the same media type.  Didn’t want to do that today, because callers assume serializer info media type is unique.